File: app/src/main/java/com/iita/akilimo/entities/LocationInfo.java
package com.iita.akilimo.entities;


import io.objectbox.annotation.Entity;
import io.objectbox.annotation.Id;
import lombok.Data;

@Data
@Entity
public class LocationInfo {

    @Id
    long id;

    public double latitude;
    public double longitude;
    public double altitude;

    public String placeName;
    public String address;

}

File: app/src/main/java/com/iita/akilimo/entities/LocationInfo.kt
package com.iita.akilimo.entities

import com.j256.ormlite.field.DatabaseField
import com.j256.ormlite.table.DatabaseTable
import io.realm.RealmObject
import io.realm.annotations.PrimaryKey

@DatabaseTable
open class LocationInfo {

    @DatabaseField(columnName = "id", generatedId = true)
    var id: Int? = null
    var latitude: Double = 0.0
    var longitude: Double = 0.0
    var altitude: Double = 0.0

    var placeName: String? = null
    var address: String? = null
}